### PR TITLE
Basic channel support and a couple small channel pages

### LINKED ISSUE #17: Channel Pages
What would be really awesome is for places like #javascript to have it's own page detailing meetups and guides, or places like #homelab detailing where to get free/cheap evaluation software/training.

Going to propose to work up some basic templates, a basic setup where a channel can have one _or possibly more_ pages. This kind of links to #16 being as I'm not sure if we want to throw all these up on the public site's menu or if we just want to keep this internal except for bots possibly crawling it.

I'm all for public visibility but I know the site design is basic and cramped right now, so it may have to wait for a design overhaul.

### PR DESCRIPTION
**Pull request by StrangeWill**
If I'm doing Jekyll wrong feel free to hit me over the head with a stick

resolves #17

### CODE REVIEW

File: _pages/channels/homelab.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+layout: channel
+title: Homelab
+channel: homelab
+permalink: /channels/homelab/
+parent: channels
+---
+
+A place for people have
```
Comment (brb3) [inline _pages/channels/homelab.md]: wait, wut?

**Comment by brb3** [comment]
Other than the weird sentence, I think this looks good to me. Can you finish your thought on that homelab page @StrangeWill , and I will go ahead and merge this. I'd like to start dumping some resources for #ham 👍

**Comment by agarzola** [comment]
ham

File: _pages/channels/homelab.md (hunk 1 above)
Comment (StrangeWill) [inline _pages/channels/homelab.md]: This was me typing something, having a brain fart, going on to other things and completely forgetting a had a half-finished sentence somewhere.

**Comment by StrangeWill** [comment]
@agarzola Go ahead and pull my branch and let me know what you want to do about the menu situation and make a PR into my branch. :D

**Comment by agarzola** [comment]
I somehow missed your comment from last night, @StrangeWill. My bad!

I did address the merge conflict, though, via Github’s interface, just to see what that’s like.

**Comment by StrangeWill** [comment]
OK I think we're ready :D

**Pull request merged**